### PR TITLE
Add validation when user creates a cluster only with no-proxy (non-interacitve mode)

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: Tzvika Avni <tavni@redhat.com>

SDA-6712

### CODE REVIEW

File: cmd/ocm/create/cluster/cmd.go [hunk 1]
```diff
@@ -688,6 +688,11 @@ func promptClusterWideProxy() error {
 			}
 		}
 	}
+	if (args.clusterWideProxy.HTTPProxy != nil && *args.clusterWideProxy.HTTPProxy == "") &&
+		(args.clusterWideProxy.HTTPSProxy != nil && *args.clusterWideProxy.HTTPSProxy == "") &&
+		*args.clusterWideProxy.NoProxy != "" {
```
Comment (zvikorn) [inline cmd/ocm/create/cluster/cmd.go:693]: args.clusterWideProxy.NoProxy cannot be nil

Comment (tbrisker) [reply cmd/ocm/create/cluster/cmd.go:693]: are you sure?
https://github.com/openshift-online/ocm-cli/pull/407/files#diff-2f19822c49dc0bec88a73920ea0f04a30b7047b5a827c90447e02b7378744f97R648

Comment (zvikorn) [reply cmd/ocm/create/cluster/cmd.go:693]: https://github.com/openshift-online/ocm-cli/blob/main/pkg/arguments/arguments.go#L224

Comment (tbrisker) [reply cmd/ocm/create/cluster/cmd.go:693]: The link didn't work as expected, but take a look at line 648 where it's being set to `nil`

Comment (zvikorn) [reply cmd/ocm/create/cluster/cmd.go:693]: Weird... I have tested all options I was thinking about. I assume when we set it with "", the length is not 0?
Anyway, I am adding the check for nil

**Pull request merged**